data/data/powervs/variables-powervs.tf

```diff
@@ -46,6 +46,45 @@ variable "cloud_instance_id" {
   default     = ""
 }
 
+################################################################
+# Configure storage
+################################################################
+variable "cos_resource_group_name" {
+  type = string
+  description = "The name of the COS resource group that holds your resource"
+  default = ""
+}
+
+variable "cos_instance_name" {
+    type        = string
+    description = "The name of the COS instance that will hold your bucket"
+    default     = ""
+}
+
+variable "cos_instance_plan" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "standard"
```

Comment:
can we set the plan to default to "power-virtual-server-group"? i'm not sure it matters, but since users will have this plan, and we set it to that for the service creation, consistency seems good to me

Reply:
There are only 2 value supported here - `lite` and `standard` for the cos instance, I think we are okay with standard because lite is supported only one instance per account(like free one)

Reply:
Done

Reply:
Actually, removing this var for now as it's unused. It was intended for when I was initially creating the COS instance via terraform. Also @mkumatag says that there are only two values for this, lite or standard, so if we wind up creating the COS instance in terraform, I'll be hardcoding it to "standard" (see @mkumatag's comment below)